bessctl/commands.py

```diff
@@ -360,6 +360,22 @@ def get_var_attrs(cli, var_token, partial_word):
             var_type = 'gate'
             var_desc = 'input gate of a module (default 0)'
 
+        elif var_token == 'GATEHOOKCLASS':
```

Comment:
Aside: I'd like to simplify all this code at some point, if there were a good way to make this more table-driven for instance. I haven't quite figured out how to do that yet though.